src/google/TreeMap.js

```diff
@@ -0,0 +1,168 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "../common/HTMLWidget", "goog!visualization,1,packages:[treemap]"], factory);
+    } else {
+        root.TreeMap = factory(root.d3, root.common_HTMLWidget);
+    }
+}(this, function (d3, HTMLWidget) {
+
+    function TreeMap() {
+        HTMLWidget.call(this);
+        
+        this._class = "google_TreeMap";
+        this._tag = "div";
+        this._chartType = "TreeMapChart";
+
+        this._data_google = [];
+        this.columns([]);
+        this.data([]);
+    }
+    TreeMap.prototype = Object.create(HTMLWidget.prototype);
```

Comment:
Like this?

+        this._class = "google_TreeMap";

+        this._tag = "div";

+        this._chartType = "TreeMapChart";

+        this._data_google = [];

+        this.columns([]);

+        this.data([]);